clients/unshaded/src/main/java/tachyon/client/block/BlockStoreContext.java

```diff
@@ -56,7 +57,7 @@
    * manner.
    */
   private synchronized void initializeLocalBlockWorkerClientPool() {
-    NetAddress localWorkerAddress =
+    WorkerNetAddress localWorkerAddress =
```

Comment:
WorkerNetAddress is a public API, and therefore better not to be a generated class.